src-v2/NTS.Domain.Core/Objects/Start.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NTS.Domain.Core.Entities.ParticipationAggregate;
+using NTS.Domain;
+using NTS.Domain.Core.Entities;
+using Not.Localization;
+
+namespace NTS.Domain.Core.Objects;
+
+public class StartList
+{
+    public StartList(IEnumerable<Participation> participations)
+    {
+        foreach (var participation in participations)
+        {
+            if (!participation.IsEliminated())
+            {
+                var phases = participation.Phases;
+                var now = new Timestamp(DateTime.Now);
+                foreach (var phase in phases.Where(p=>p.StartTime!=null))
+                {
+                    var index = phases.IndexOf(phase);
+                    if (now - phase.StartTime > TimeSpan.FromMinutes(15))
+                    {
+                        var previousStart = new Start(participation.Combination.Name, participation.Combination.Number, index + 1, phases[index].Length, participation.Phases.Distance, phase.StartTime!);
+                        PreviousStarts.Add(previousStart);
+                    }
+                    else
+                    {
+                        var upcomingStart = new Start(participation.Combination.Name, participation.Combination.Number, index + 1, phases[index].Length, participation.Phases.Distance, phase.StartTime!);
```

Comment:
This instantiation is an exact copy of of the previous one. What are we doing with copy pase code?

Reply:
Furthermore it's very hard to read. It isn't at all obvious what are you using index for.